doc : Add note in SpringBootHealthCheckEnricher documentation about enabled management health probe endpoints (#3146)
eclipse-jkube/jkube: jkube-kit/doc/src/main/asciidoc/inc/enricher/spring-boot-healthcheck/_jkube_healthcheck_spring_boot.adoc

The enricher will try to discover the settings from the `application.properties` / `application.yaml` Spring Boot configuration file.

> `/actuator/health` is default endpoint for liveness and readiness probes. If user has enabled `management.health.probes.enabled` property this enricher would add `/actuator/health/liveness` as liveness and `/actuator/health/readiness` as readiness probe endpoints.

Reviewer: ```suggestion
`/actuator/health` is the default endpoint for the liveness and readiness probes.
If the user has enabled the `management.health.probes.enabled` property this Enricher uses the `/actuator/health/liveness` as liveness and `/actuator/health/readiness` as readiness probe endpoints instead.
```
